KPI/E.adoc

```diff
@@ -1,106 +1,147 @@
+
+```xml
+<gmd:identificationInfo>
+  ...
+  <gmd:temporalElement>
+     <gmd:EX_TemporalExtent id="temporalExtent">
+       <gmd:extent>
+         <gml:TimePeriod gml:id="boundingTemporalExtentPeriod">
+           <gml:beginPosition>2005-10-01</gml:beginPosition>
+           <gml:endPosition indeterminatePosition="now"/>
+         </gml:TimePeriod>
+       </gmd:extent>
+     </gmd:EX_TemporalExtent>
+  </gmd:temporalElement>
+</gmd:temportalElement>
+```
+
+The Data Update Frequency is defined in `gmd:maintenanceAndUpdateFrequency`:
+
+Below is an example of a GTS bulletin whic is updated every 6 hours.
+
+```xml
+<gmd:identificationInfo>
+  ...
+  <gmd:resourceMaintenance>
+    <gmd:MD_MaintenanceInformation>
+      <gmd:maintenanceAndUpdateFrequency>
+        <gmd:MD_MaintenanceFrequencyCode codeListValue="irregular" codeList="http://standards.iso.org/ittf/PubliclyAvailableStandards/ISO_19139_Schemas/resources/codelist/gmxCodelists.xml#MD_MaintenanceFrequencyCode"/>
+      </gmd:maintenanceAndUpdateFrequency>
+      <gmd:userDefinedMaintenanceFrequency>
+        <gts:TM_PeriodDuration>PT6H</gts:TM_PeriodDuration>
+      </gmd:userDefinedMaintenanceFrequency>
+      <gmd:maintenanceNote>
+        <!-- DRAFT – Guidance on WIS Discovery Metadata (following CBS-16), p38 -->
+        <gco:CharacterString>Instances of bulletin SIKB20NGTT are available every 6 hours starting at 03 UTC.</gco:CharacterString>
+      </gmd:maintenanceNote>
+    </gmd:MD_MaintenanceInformation>
+  </gmd:resourceMaintenance>
+  ...
+</gmd:identificationInfo>
+```
+
+If it is not relevant or necessary to provide information regarding the data
+update frequency, `gmd:MD_MaintenanceFrequencyCode` can be set to `asNeeded`:
+
+```xml
+<gmd:identificationInfo>
+  ...
+  <gmd:resourceMaintenance>
+      <gmd:MD_MaintenanceInformation>
+        <gmd:maintenanceAndUpdateFrequency>
+          <gmd:MD_MaintenanceFrequencyCode codeList="http://standards.iso.org/iso/19139/resources/gmxCodelists.xml#MD_MaintenanceFrequencyCode" codeListValue="asNeeded"/>
+        </gmd:maintenanceAndUpdateFrequency>
+      </gmd:MD_MaintenanceInformation>
+  </gmd:resourceMaintenance>
+  ...
+</gmd:identificationInfo>
+```
+
+It is also possible to identify a dataset status / progress using the `MD_ProgressCode`
+codelist.  The following example provides an ongoing dataset progress/status:
+
+```xml
+<gmd:identificationInfo>
+  ...
+  <gmd:status>
+    <gmd:MD_ProgressCode codeList="http://www.isotc211.org/2005/resources/Codelist/gmxCodelists.xml#MD_ProgressCode" codeSpace="ISOTC211/19115" codeListValue="onGoing">onGoing</gmd:MD_ProgressCode>
+  </gmd:status>
+</gmd:identificationInfo>
+```
+
+.Temporal information implementation rules
+
+|===
+|Rule |Score
+
+|The Temporal Extent is defined
+|1
+
+|The Temporal Extent contains begin and end components
+|1
+
+|The begin date time is less than or equal to the end date time
+|1
+
+|The Data Update Frequency is defined
+|1
+
+|The Data Status is defined
```

Comment:
Is the "gmd:status" obligatory or is this only to motivate people to use it?

Reply:
as discussed, not obligatory.